File: Basic/code/Problem_8.java
// 8. Input temperature in Fahrenheit and convert into Celsius.


import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
		 Scanner sc= new Scanner(System.in);  
       System.out.println("Enter temperature in Fahrenheit - ");  
        
       float fahrenheit= sc.nextInt(); 
       System.out.println("temperature in Fahrenheit : "+fahrenheit+"°F"); 
       ConvertFahrenheitToCelsius(fahrenheit);
	}
	private static void ConvertFahrenheitToCelsius(float tempF){
	   
	    float celsius = ((tempF-32) * 5/9);
	    System.out.println("temperature in celsius : "+celsius+"°C");
	
	}
}

File: Basic/code/problem_8.c
/*
Write a program to Input temperature in Fahrenheit and convert into Celsius.
*/

// Solved by napalion

#include<stdio.h>
int main(){
    float Fahrenheit,Celsius;
    printf("Enter the temperature in Fahrenheit: \n");
    scanf("%f",&Fahrenheit);
    printf("The temperature %f°F = %f°C\n",Fahrenheit,((Fahrenheit-32)*5)/9);
    return 0;
}